### PR TITLE
✨ Source Apple Search Ads : Migrate to Manifest-only

### PR DESCRIPTION
**Pull request by topefolorunso**
## What

Migrate Source Apple Search Ads to Manifest-only

## Review guide

1. `manifest.yaml`

## Can this PR be safely reverted and rolled back?

- [x] YES 💚
- [ ] NO ❌

### CODE REVIEW

**Comment by natikgadzhi** [comment]
Regression tests refuse to run ;( they're not failing, they just didn't run. Same thing that is blocking @pabloescoder on a few PRs. I'll try to get to this this week.

File: airbyte-integrations/connectors/source-apple-search-ads/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,1135 @@
...
+    campaigns_report_daily: false
+    adgroups_report_daily: false
+    keywords_report_daily: false
+  yamlComponents:
+    streams:
+      keywords_report_daily:
+        - errorHandler
+    global:
+      - authenticator
+  testedStreams:
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-apple-search-ads/manifest.yaml:532]: @topefolorunso did you actually have test data to verify these? Not copy and pastedfrom another connector, right? ;)

**Review by natikgadzhi — APPROVED** [review]
Overall, if the streams are tested, I would be down to merge this. Just confirming they are.

**Comment by topefolorunso** [comment]
No not tested yet. Still waiting on @ChristoGrab for creds

File: airbyte-integrations/connectors/source-apple-search-ads/manifest.yaml (hunk 1 above)
Comment (topefolorunso) [reply airbyte-integrations/connectors/source-apple-search-ads/manifest.yaml:532]: No not tested yet. Still waiting on @ChristoGrab for creds

Comment (ChristoGrab) [reply airbyte-integrations/connectors/source-apple-search-ads/manifest.yaml:532]: Looking into this now. We have an existing account in LastPass, but no Oauth creds. [Working through the steps to generate them].(https://developer.apple.com/documentation/apple_search_ads/implementing_oauth_for_the_apple_search_ads_api)

**Comment by natikgadzhi** [comment]
Yolo, let's move forward. I'll revert if we see any problems with this, and ask @topefolorunso to fix them.

**Comment by olagjo** [comment]
> Yolo, let's move forward. I'll revert if we see any problems with this

I think there were some problems with this 🤓 https://github.com/airbytehq/airbyte/issues/46879
(However, instead of reverting, the problems are quite easy to fix by continuing to move forward: https://github.com/airbytehq/airbyte/pull/46706)

**Pull request merged**